src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc

```diff
@@ -207,10 +207,16 @@ bool AlertManeuverRequest::PrepareResponseParameters(
     return_info = std::string("Unsupported phoneme type sent in a prompt");
     return result;
   }
+
   result_code =
       PrepareResultCodeForResponse(navigation_alert_info, tts_alert_info);
   return_info =
       MergeInfos(navigation_alert_info, info_navi_, tts_alert_info, info_tts_);
+
+  if (mobile_apis::Result::UNSUPPORTED_RESOURCE == result_code) {
+    // According to APPLINK-19582 should be success:true in this case
+    return true;
+  }
```

Comment:
@dev-gh thanks for good idea, but in this case it's impossible to move this logic exactly to `PrepareResultForMobileResponse/PrepareResultCodeForResponse` as you suggest. First one is const (can't modify `info_tts_` class member to pass new string message) and second one is with const parameters (can't modify `result_code` property to pass overriden result code). Also this logic depends from result of both mentioned functions so I just created a new separate function and move all additional logic and checks to it. Please see https://github.com/smartdevicelink/sdl_core/pull/1567/commits/efb2f68675088363bbe0bff96e1833fe982fbcb1